### PR TITLE
Styles inputs in group modal to be consistent with rest of app #9897

### LINKED ISSUE #9897: Plan group modal issue
[//]: # (Before logging this issue, please post to the Report a Bug guild from the Habitica website's Help menu. Most bugs can be handled quickly there. If a GitHub issue is needed, you will be advised of that by a moderator or staff member -- a player with a dark blue or purple name. It is recommended that you don't create a new issue unless advised to.)

[//]: # (Bugs in the mobile apps can also be reported there.)

[//]: # (If you have a feature request, use "Help > Request a Feature", not GitHub or the Report a Bug guild.)

[//]: # (For more guidelines see https://github.com/HabitRPG/habitica/issues/2760)

[//]: # (Fill out relevant information - UUID is found from the Habitia website at User Icon > Settings > API)
### General Info
* UUID: 2d57fd14-0fd1-4bf5-abe9-d65bb42fbc6f
* Browser: Google Chrome 63.0.3239.132
* OS: Ubuntu 17.04

### Description
[//]: # (Describe bug in detail here. Include screenshots if helpful.)

Upon a click on the "Create Your New Group" button on [this](https://habitica.com/group-plans) page, a modal window opens and the page scrolls to the very top (should not it be?)

image

And we need to stylize checkboxes and radio buttons, now they are displayed in the usual way (browser 
 rendering).

image

#### Console Errors
[//]: # (Include any JavaScript console errors here.)

### PR DESCRIPTION
**Pull request by Kiwilicious**
Fixes #9897 
screenshot from 2018-02-02 01-09-22

### CODE REVIEW

File: website/client/components/groups/groupPlan.vue [hunk 1]
```diff
@@ -86,16 +86,13 @@ div
...
+        .custom-control.custom-radio
+          input.custom-control-input(type="radio", name="new-group-privacy", value="private", v-model="newGroup.privacy")
+          label.custom-control-label {{ $t('inviteOnly') }}
       .form-group
-        .checkbox
-          label
-            input(type='checkbox', v-model='newGroup.leaderOnly.challenges')
-            | {{ $t('leaderOnlyChallenges') }}
+        .custom-control.custom-checkbox
+          input.custom-control-input(type="checkbox", v-model="newGroup.leaderOnly.challenges" :id="_uid")
```
Comment (paglias) [inline website/client/components/groups/groupPlan.vue]: what's uid here? using it with `:attr` indicates a Vue variable but I can't find any with this name. Just give it an unique id like `id="create-group-leaderOnlyChallenges-checkbox"`

**Review by paglias — DISMISSED** [review]

File: website/client/components/groups/groupPlan.vue (hunk 1 above)
Comment (Kiwilicious) [reply website/client/components/groups/groupPlan.vue]: _uid refers to the component's unique id. https://stackoverflow.com/questions/34950867/vue-js-how-to-set-a-unique-id-for-each-component-instance. However looking back on it, putting the component's id on a single element might be a bad idea. I'll change it right now.

**Comment by paglias** [comment]
Working great :) It'll get merged around monday and should go live by the end of next week, thanks!

**Comment by SabreCat** [comment]
Thanks @Kiwilicious! Noted toward your second contributor tier!

**Pull request merged**